.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "css.validate": false,
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "files.insertFinalNewline": true,
+  "files.trimTrailingWhitespace": true,
```

Comment:
these seem to overlap with editorconfig settings, yes?

https://github.com/nodejs/nodejs.org/blob/main/.editorconfig#L8-L9

Reply:
Feel free to remove them (I can also do it)